### DISCUSSION TITLE
How to enable filtering on selection type item itself

### DISCUSSION THREAD

**Discussion by allenlim-0627** [bosskmk/pluto_grid#274] [Q&A]
Hi bosskmk,

I have a question regarding on Selection type item. Currently I see that I can enable filter menu item on pluto column by setting the value of "enableFilterMenuItem" of "PlutoColumn" object so we can filter out pluto grid based on the value I set on the filter.

But what if when I select the selection type item. it will shows a pop up grid contains of selection items and I want to do searching from the list of selection item. Is there a way for me to customize the pop up grid so that I can enable filtering on the pop up before I select value that I want?

Thank you so much.

**Comment by bosskmk** [comment]
Updated PlutoGrid version 2.5.0.
So that you can filter on the select column you want.
See the link below.

Demo
https://weblaze.dev/pluto_grid/build/web/#feature/selection-type-column

Demo source
https://github.com/bosskmk/pluto_grid/blob/master/example/lib/screen/feature/selection_type_column_screen.dart
